binrz/man/rz-asm.1

```diff
@@ -1,108 +1,78 @@
-.Dd Oct 30, 2018
+.Dd Jan 22, 2024
 .Dt RZ_ASM 1
 .Sh NAME
 .Nm rz-asm
 .Nd rizin assembler and disassembler tool
 .Sh SYNOPSIS
 .Nm rz-asm
-.Op Fl ABdDeEfCLvwrqx
+.Op Fl ACdDehLBvw 
```

Comment:
what's the reason to remove some of the options?

Reply:
it must be wrong in `rz-asm -h` making a commit changing that too , should all the options be present in this?

Reply:
I'm not sure tbh... @XVilka ?! do you know if man pages usually have all options listed here?

Reply:
Yes, man concatenates those with variables after that string, just run `man rz-asm` and see how it looks.